superset-frontend/src/common/components/Modal/Modal.tsx

```diff
@@ -105,7 +105,9 @@ const StyledModal = styled(BaseModal)<StyledModalProps>`
   }
 
   // styling for Tabs component
-  .ant-tabs {
+  // Aaron note 20-11-19: this seems to be exclusively here for the Edit Database modal.
+  // TODO: remove this as it is a special case.
```

Comment:
Maybe we should do the TODO in this PR?

Reply:
I disagree, not that relevant to this PR